### PR TITLE
doc/releases: fix formatting for the 17.2.8 release (el8 CERN)

### PR DESCRIPTION
**Pull request by ebocchi**
Signed-off-by: Enrico Bocchi <enrico.bocchi@cern.ch>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Review by Svelar — APPROVED** [review]

**Comment by anthonyeleven** [comment]
jenkins retest this please
I filled in the checklist

**Comment by ljflores** [comment]
https://ceph--61002.org.readthedocs.build/en/61002/releases/quincy/

File: doc/releases/quincy.rst [hunk 1]
```diff
@@ -26,10 +26,10 @@ from https://docs.ceph.com/en/latest/install/get-packages/#rhel, the ceph.repo f
...
 
 These CERN packages come with no warranty and have not been tested. The software in them has been
-tested by Ceph according to [platforms](https://docs.ceph.com/en/latest/start/os-recommendations/#platforms).
-The repository for el8 builds is hosted by CERN on [Linux @ CERN](https://linuxsoft.cern.ch/repos/ceph-ext-quincy8el-stable/).
-The public part of the GPG key used to sign the packages is available at
-[RPM-GPG-KEY-Ceph-Community](https://linuxsoft.cern.ch/repos/RPM-GPG-KEY-Ceph-Community).
+tested by Ceph according to https://docs.ceph.com/en/latest/start/os-recommendations/#platforms
+The repository for el8 builds is hosted by CERN on Linux@CERN: https://linuxsoft.cern.ch/repos/ceph-ext-quincy8el-stable/
+The public part of the GPG key ('RPM-GPG-KEY-Ceph-Community') used to sign the
+packages is available at https://linuxsoft.cern.ch/repos/RPM-GPG-KEY-Ceph-Community
```
Comment (ljflores) [inline doc/releases/quincy.rst:32]:
```suggestion
tested by Ceph according to `platforms <https://docs.ceph.com/en/latest/start/os-recommendations/#platforms>`_ .
The repository for el8 builds is hosted by CERN on `Linux@CERN <https://linuxsoft.cern.ch/repos/ceph-ext-quincy8el-stable/>`_ .
The public part of the GPG key used to sign the
packages is available at `RPM-GPG-KEY-Ceph-Community <https://linuxsoft.cern.ch/repos/RPM-GPG-KEY-Ceph-Community>`_ .
```

**Review by ljflores — COMMENTED** [review]
Try using this format for embedded links.

**Review by ljflores — APPROVED** [review]

**Pull request merged**